integration_tests/README.rst

```diff
@@ -6,9 +6,10 @@ a different directory.
 
 You can invoke these tests with the command:
 
-    pytest tests/integration --org <orgname>
+    pytest integration_tests/integration --org <orgname>
 
-You can leave out the ``--org`` option if you have an org defaulted.
+--org should be a garbage org that you do not care about because it may be
+      changed.
```

Comment:
Should we add an assertion somewhere that it is a scratch org?

Reply:
https://github.com/SFDO-Tooling/CumulusCI/pull/1816/commits/e1bdf07bc3899f6d19b51a13bf5c997503dc9c1d#diff-20c920c476d998d34927d826ade54c1dR38